### DISCUSSION TITLE
Portal and its entities - Visibility

### DISCUSSION THREAD

**Discussion by psundarkyndryl** [apache/airflow#24555] [General]
### Description

Below features would be useful in certain scenarios:

1.  A configuration in the airflow.cfg to show or suppress the UI(portal)
2.  A parameter in the config file to show or suppress the conf of the dag_run in the portal.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Just to set some expectations.

You need to be more specific what you think about by disabling the portal and dag_run. And start a real discussion about it.

This is an open-source project with > 2000 contributors and you can become one. Oening a feature will not "magically" make it implemented. If you really want this to be done, cannot do it yoursekve you need to convince someone that you want makes sense, explain why, and justify the increased complexity it brings. If you don't do it, this feature will just stay open with no-one even looking at it. And the BEST you can do is to draft a proposal yourself what you think might be a good solution (in the form of PR and discuss it there).

What you opened is a wishlist without any attempt to explain what you want, so it's unlikely anyone will do anything about it.

I am converting it into discussion until any of those happens.
